src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -0,0 +1,108 @@
+
+import meshio
+from meshio._mesh import CellBlock
+import numpy as np
+import argparse
+
+
+def convert_abaqus_to_gmsh(input_mesh, output_mesh):
+  """
+  @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information
+  @param input_mesh path of the input abaqus file
+  @param output_mesh path of the output gmsh file
+  """
+  # Load the mesh
+  print('Reading abaqus mesh...')
+  mesh = meshio.Mesh.read(input_mesh, "abaqus")
+
+  # Add a default region tag
+  print('Converting region tags...')
+  region_list = list(mesh.cell_sets.keys())
+  Nregion = len(region_list)
+  cell_ids = np.zeros(len(mesh.cells[0][1]), dtype=int)
+  for ii, ka in zip(range(Nregion), region_list):
```

Comment:
Please use the `enumerate` feature instead (_in the whole codebase_).
Also, why do you use `ii` instead of more readable `i`, `idx`, `region_idx`... ?
What does `ka` stands for?

Reply:
@TotoGaz - Sure, I can be clearer here.  

These are just my preferred counter (ii, jj, kk, doubled since matlab used to complain about single i's and j's) and nested key names (ka, kb, kc, ...) when I'm being lazy.